### PR TITLE
Add basic tests to reference package

### PR DESCRIPTION
**Pull request by armandfardeau**
Add basic tests for initial setup
Further tests are going to be added

### CODE REVIEW

File: .rubocop_rails.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+require: rubocop-rails
+
+Rails:
+  Enabled: true
+
+Rails/ActionFilter:
+  Include:
+    - decidim-*/app/controllers/**/*.rb
```
Comment (Quentinchampenois) [inline .rubocop_rails.yml:8]: Hi @armandfardeau,

Here we should adapt the rubocop rules to the osp-app architecture

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]

File: .rubocop_rails.yml (hunk 1 above)
Comment (armandfardeau) [reply .rubocop_rails.yml]: Thanks @Quentinchampenois. FIxed it

File: config/locales/fr.yml [hunk 2]
```diff
@@ -1,40 +1,68 @@
...
       osp_authorization_handler:
+        birthday: Date de naissance
         document_number: Numéro unique
         postal_code: Code postal
-        birthday: Date de naissance
+      participatory_process:
+        private_space: Espace privé
   decidim:
+    accessibility:
+      skip_button: Skip button
```
Comment (MoretS) [inline config/locales/fr.yml:13]: @armandfardeau It should be "Passer au contenu principal"

File: config/locales/fr.yml [hunk 3]
```diff
@@ -1,40 +1,68 @@
...
+      - FR9
   layouts:
     decidim:
       footer:
         download_open_data: Données ouvertes
         made_with_open_source: Site réalisé par <a target="_blank" href="https://opensourcepolitics.eu">Open Source Politics</a> grâce au <a target="_blank" href="https://github.com/decidim/decidim">logiciel libre Decidim</a>.
-  faker:
-    address:
-      country_code: ['FR','FR0','FR1','FR2','FR3','FR4','FR5','FR6','FR7','FR8','FR9']
+      header:
```
Comment (MoretS) [inline config/locales/fr.yml:60]: @armandfardeau There are still English translations in the French translation file

**Review by MoretS — CHANGES_REQUESTED** [review]
Thanks @armandfardeau!

**Review by Quentinchampenois — APPROVED** [review]

**Review by MoretS — CHANGES_REQUESTED** [review]
Hi @armandfardeau! Thanks for your work

Still english translations in fr locale file and french translations in en locale file

**Review by MoretS — APPROVED** [review]

**Pull request merged**